packages/fxa-settings/src/components/ButtonDownloadRecoveryKeyPDF/RecoveryKeyPDF.tsx

```diff
@@ -50,8 +50,8 @@ export const RecoveryKeyPDF = ({
     fonts: [
       {
         src: notoSansMonoSemiBold,
-        style: 'normal',
-        weight: 'semibold',
+        fontStyle: 'normal',
+        fontWeight: 'semibold',
```

Comment:
cc @vpomerleau

Reply:
Thanks for fixing! The previous package versions must have been validating more loosely and this was missed. FWIW, I tested before and after this change, the PDF result is the same. `fontStyle` and `fontWeight` might be unnecessary here.

Reply:
Thanks for testing @vpomerleau !